### PR TITLE
Fix minor typo in comment in rsa_st

### PR DESCRIPTION
**Pull request by danbev**

### CODE REVIEW

File: crypto/rsa/rsa_locl.h [hunk 1]
```diff
@@ -28,7 +28,7 @@ DEFINE_STACK_OF(RSA_PRIME_INFO)
 struct rsa_st {
     /*
      * The first parameter is used to pickup errors where this is passed
-     * instead of aEVP_PKEY, it is set to 0
+     * instead of a EVP_PKEY, it is set to 0
```
Comment (mattcaswell) [inline crypto/rsa/rsa_locl.h]: personally I would say "an EVP_PKEY"

**Comment by mattcaswell** [comment]
This looks trivial to me (from a copyright perspective). If you agree please update your commit to add "CLA: trivial" into your commit description.

File: crypto/rsa/rsa_locl.h (hunk 1 above)
Comment (danbev) [reply crypto/rsa/rsa_locl.h]: Yeah, that does sound better. I've updated this now. Thanks

**Comment by mattcaswell** [comment]
Pushed. Thanks.

**Pull request closed without merging**